static/js/experience-initializer.js

```diff
@@ -0,0 +1,60 @@
+import updateRuntimeConfig from './utils/update-runtime-config';
+
+/**
+ * Responsible for initializing the answers experience
+ */
+export default class ExperienceInitializer {
+  constructor (initializeManually) {
+    this._initializeManually = initializeManually;
+    this._hasAnswersInitialized = false;
+  }
+
+  /**
+   * Initializes on DOM load, or puts an init function on the window if initializeManually is true
+   */
+  setup () {
+    if (this._initializeManually) {
+      window.initAnswersExperience = this._generateInitFunction();
```

Comment:
Does the iFrame integration end up invoking this as well?

Reply:
Yes, if the RuntimeConfigReceiver gets a message from the parent with initAnswersFrame set to true, then it will call the same initAnswersExperience function